plugin-tester-java/src/main/resources/certs/README.md

```diff
@@ -0,0 +1,22 @@
+# Some notes about the certs in this directory
+
+Self-signing sample CA in rootCA.*, CA cert secret: "secret"
+Server cert for localhost signed by rootCA in localhost-server.*, no password for private key
+Client cert for a client to connect in localhost-client.*, no password for private key
+
+Certs used by `MtlsGreeterServer`.
```

Comment:
For reference, should we include the commands for how the test certs were created?

Reply:
[16c640a](https://github.com/akka/akka-grpc/pull/1781/commits/16c640ace8f8a3ec76e2fb7814d8aea9a5c3bfa5)